docs/api.md

```diff
@@ -65,6 +65,40 @@ when a VSP is closed will result in an error.
 **Registering a ticket is a two step process. The VSP will not add a ticket to
 its voting wallets unless both of these calls have succeeded.**
 
+#### Step Zero (optional)
+
+Set an alternate signing address for a ticket. The ticket must be valid and will
+be transmitted to the network if not found. If set, the vsp will check that a
+signature is good for this address and fallback to the commitment address if not.
+The address must be valid for the network and a pay to secp256k1 ecdsa pubkey
+hash script. Can be cleared by sending `""` as the `altpubaddress`. The key may
+be set a maximum of three times. Further requests to set a new key will be
```

Comment:
s / `The key may be set a maximum of three times.` / `The key currently can only be set once, users are advised to crosscheck inputs as many times as necessary.`

Reply:
ok